netket/hilbert/continuous_hilbert.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, domain: Tuple[float, ...], pbc: Union[bool, Tuple[bool, ...]]
 
         Args:
             domain: range of the continuous quantum numbers
+            pbc: Whether or not to use periodic boundary conditions
         """
```

Comment:
Can you write more stuff about those two docstrings? Something Like *(but written better)*
 - domain: upper boundaries of the allowed values in the configurations (?). Assumes that the lower bound is always 0. If `np.inf` is used an infinite box is considered. It should be provided as an iterable where every element corresponds to a different dimension. 
 -  pbc: an utterable of the same length of domain specifying whether the corresponding dimension should be considered periodic or not.